workflow/workflow.go

```diff
@@ -385,5 +385,38 @@ func NewWorkflowCommands() []*cli.Command {
 				}),
 			Action: TraceWorkflow,
 		},
+		{
+			Name:      "update",
+			Usage:     common.UpdateWorkflowDefinition,
+			UsageText: common.WorkflowUpdateUsageText,
+			Flags: append(common.FlagsForExecution,
+				&cli.StringFlag{
+					Name:     common.FlagUpdateWaitPolicy,
+					Usage:    "Wait policy determines how long the client should wait for a return value from the server:" + strings.Join(mapKeysToArray(updateWaitPolicyMap), ", "),
```

Comment:
It's not "how long", it's what to wait for.
And FTR currently only COMPLETED is implemented.